models/public/aclnet/aclnet.md

```diff
@@ -0,0 +1,68 @@
+# aclnet
+
+## Use Case and High-Level Description
+
+The `AclNet` model is designed to perform sound classification.
+The `AclNet` model is trained on a dataset of environmental sounds (DES-53).
+For details about the model, see this [paper](https://arxiv.org/pdf/1811.06669.pdf).
+
+The model input is a segment of PCM audio samples in [N, C, H, W] format with the shape [1, 1, 1, 16000].
+
+The model output for `AclNet` is the sound classifier output for the 53 different environmental sound classes from the DES-53 database.
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 1.4           |
+| MParams           | 2.7           |
+| Source framework  | PyTorch\*     |
+
+## Accuracy
+
+See this [publication](http://dcase.community/documents/workshop2019/proceedings/DCASE2019Workshop_Huang_52.pdf) and this [paper](https://arxiv.org/pdf/1811.06669.pdf).
+
+## Performance
+
+## Input
+
+### Original Model
+
+Audio, name - `0`, shape - `1,1,1,16000`, format is `N,C,H,W` where:
+
+- `N` - number of samples
```

Comment:
Shouldn't this be the batch size? It seems like it's the last dimension that is the number of samples.

Reply:
Yes batch size is more accurate. changed it to avoid confusion.